hacks/weapons/custom/shotgun.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<hack name="Shotgun Hack" author="Noxid">
+	<panel>
+		<field type="label">REPLACES MACHINE GUN</field>
+		<field type="info">
+		This turns the Machine Gun and the MG bullets to become a shotgun. Knockback
+		is applied to the player when fired. Refer to /hacks/bitmap/shotgun_example.bmp
```

Comment:
i should add an "image" element to the xml structure... but not today :3

Reply:
that would help with overcoming the weird /bitmap/ folder thing.